app/controllers/friends_controller.rb

```diff
@@ -0,0 +1,19 @@
+class FriendsController < ApplicationController
+
+  def create
+    friend = User.find_by(email: params[:friend_email])
+    if friend == nil
+      flash[:error] = 'Error, friend does not have an account'
+    elsif friend.email == current_user.email
+      flash[:error] = 'Error, cannot friend yourself'
+    else
+      friend_add = Friend.new(friender: current_user, friendee: friend)
+      if friend_add.save
+        flash[:error] = 'Friend Added!'
```

Comment:
do we want this to be a flash[:error]? i mean, it works alright... but naming a successful transaction under 'error' feels weird